HTTPServer.test/ServerTests.cs

```diff
@@ -7,11 +7,34 @@ namespace HTTPServer.test
 {
     public class ServerUnitTests
     {
+        Server server = new Server();
+
         [Fact]
         public void ServerCanStart()
         {
-            var server = new Server();
-            Assert.True(server.Start() && server.running == false);
+            Assert.True(server.Start() && server.Running);
+        }
+
+        [Fact]
+        public void ServerCanStop()
+        {
+            server.Start();
+            Assert.True(server.Stop() && !server.Running);
+        }
+
+        [Fact]
+        public void ServerCanRespond200()
+        {
+            FileConnection file = new FileConnection();
```

Comment:
Why use files in the tests?  Once you have an abstraction, you're free to implement it however is convenient for the scenario at hand.